pkg/services/ngalert/models.go

```diff
@@ -0,0 +1,92 @@
+package ngalert
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/ngalert/eval"
+)
+
+// AlertDefinition is the model for alert definitions in Alerting NG.
+type AlertDefinition struct {
+	Id        int64
+	OrgId     int64
+	Name      string
+	Condition string
+	Data      []AlertQuery
+}
+
+var (
+	// ErrAlertDefinitionNotFound is an error for an unknown alert definition.
+	ErrAlertDefinitionNotFound = fmt.Errorf("could not find alert definition")
```

Comment:
This doesn't need exporting.
```suggestion
	// errAlertDefinitionNotFound is an error for an unknown alert definition.
	errAlertDefinitionNotFound = fmt.Errorf("could not find alert definition")
```